test/parallel/test-repl-.paste.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+const terminalCode = '\u001b[1G\u001b[0J> \u001b[3G';
+
+function run(input, output, event) {
+  const stream = new common.ArrayStream();
+  let found = '';
+
+  stream.write = (msg) => found += msg.replace('\r', '');
+
+  const expected = `${terminalCode}.paste
+// Entering paste mode (^D to finish, ^C to cancel)
+${input}${output}
+${terminalCode}`;
+
+  const replServer = repl.start({
+    prompt: '> ',
+    terminal: true,
+    input: stream,
+    output: stream,
+    useColors: false
+  });
+
+  stream.emit('data', '.paste\n');
+  stream.emit('data', input);
+  replServer.write('', event);
+  replServer.close();
+  assert(found === expected, `Expected: ${expected}, Found: ${found}`);
+}
+
+const tests = [
+  {
+    input: '',
+    output: '\n(To exit, press ^C again or type .exit)',
+    event: { ctrl: true, name: 'c'}
```

Comment:
Can you remove the extra space after `{` here and on the next few similar lines.